From e9079f85fa94147433af273bc5ce90d52c7040c1 Mon Sep 17 00:00:00 2001
From: Vinayakumar B <vinayakumarb@apache.org>
Date: Fri, 8 May 2015 10:22:11 +0530
Subject: [PATCH 1812/2848] HADOOP-11922. Misspelling of threshold in
 log4j.properties for tests in hadoop-tools
 (Contributed by Gabor Liptak)

(cherry picked from commit f33efbc82709e75d9b4c4ed9835629668dbade7d)
(cherry picked from commit 14d88a85e490cd844995b9d8daeff78752dca9c1)

Change-Id: If6cbc3e6baf0343700f79a46ab38a84ffd22f66a
---
 .../hadoop-aws/src/test/resources/log4j.properties |    2 +-
 .../src/test/resources/log4j.properties            |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-tools/hadoop-aws/src/test/resources/log4j.properties b/hadoop-tools/hadoop-aws/src/test/resources/log4j.properties
index 1a6baae..ced0687 100644
--- a/hadoop-tools/hadoop-aws/src/test/resources/log4j.properties
+++ b/hadoop-tools/hadoop-aws/src/test/resources/log4j.properties
@@ -12,7 +12,7 @@
 # log4j configuration used during build and unit tests
 
 log4j.rootLogger=info,stdout
-log4j.threshhold=ALL
+log4j.threshold=ALL
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M(%L)) - %m%n
diff --git a/hadoop-tools/hadoop-azure/src/test/resources/log4j.properties b/hadoop-tools/hadoop-azure/src/test/resources/log4j.properties
index 81b935b..bb5cbe5 100644
--- a/hadoop-tools/hadoop-azure/src/test/resources/log4j.properties
+++ b/hadoop-tools/hadoop-azure/src/test/resources/log4j.properties
@@ -17,7 +17,7 @@
 # log4j configuration used during build and unit tests
 
 log4j.rootLogger=INFO,stdout
-log4j.threshhold=ALL
+log4j.threshold=ALL
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M(%L)) - %m%n
-- 
1.7.9.5

